protocol/avdl/stellar1/local.avdl

```diff
@@ -471,9 +471,13 @@ protocol local {
   KeybaseRequestID makeRequestLocal(int sessionID, string recipient, union { null, Asset } asset,
     union { null, OutsideCurrencyCode } currency, string amount, string note);
 
-  void setAccountMobileOnlyLocal(int sessionID, AccountID accountID);
-  void setAccountAllDevicesLocal(int sessionID, AccountID accountID); // opposite of mobile-only
-  boolean isAccountMobileOnlyLocal(int sessionID, AccountID accountID);
+  record AccountMobileOnly {
+    AccountID accountID;
+    boolean enabled;
+  }
+  AccountMobileOnly setAccountMobileOnlyLocal(int sessionID, AccountID accountID);
+  AccountMobileOnly setAccountAllDevicesLocal(int sessionID, AccountID accountID); // opposite of mobile-only
+  AccountMobileOnly isAccountMobileOnlyLocal(int sessionID, AccountID accountID);
```

Comment:
I think booleans would be preferable unless you've got reason to expect more return values. The frontend knows the accountID it passed in. Reflecting it by hand seems more likely to have a bug.

Reply:
this is because i thought it would be weird to get a `false` back from a method where you're changing something and it actually worked. i guess we also have the error object and no one _should_ make this mistake.

Reply:
Good point, better return void

Reply:
> better return void for the setters

lol. but that's what this whole chunk of work was about!

Reply:
yeah... sorry. Do you agree?

Reply:
hahaha. well, given the speedup and reduced communication, i guess i think it's worth it as a followon optimization like how we're doing it. on the whole though, i'm definitely a fan of separating command and query methods. i could go either way. 🤷‍♂

Reply:
discussed. i will set these back to `void`

Reply:
https://www.youtube.com/watch?v=KSSEzWXqGKY